pds_github_util/issues/move_issues.py

```diff
@@ -5,6 +5,17 @@
 import github3
 from datetime import datetime
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+
+# Quiet github3 logging
+logger = logging.getLogger('github3')
+logger.setLevel(level=logging.WARNING)
+
+>>>>>>> add script to move issue from one repository to another
```

Comment:
@tloubrieu-jpl looks like we have some merge conflicts 😢

Reply:
Yes this is a mess here

Reply:
I removed the conflicts